renovate.json

```diff
@@ -114,17 +114,20 @@
     }
   ],
   "assignees": [
-    "kashifest"
-  ],
-  "assigneesSampleSize": 1,
-  "reviewers": [
+    "elfosardo",
```

Comment:
Should we drop assignees/reviewers completely, since we use Blunderbuss for reviews these days? Renovate's lists were added when we did not have blunderbuss...

Reply:
oh yes, true  . Removing it

Reply:
Done